melodic/distribution.yaml

```diff
@@ -11638,6 +11638,21 @@ repositories:
       url: https://github.com/uos/uos_tools.git
       version: master
     status: maintained
+  ur_msgs:
+    doc:
+      type: git
+      url: https://github.com/ros-industrial/ur_msgs.git
+      version: melodic
```

Comment:
Do you mean to have a different doc branch (melodic) than the source branch (melodic-devel)?

Reply:
Yes. Documentation is built from the release branch.